### PR TITLE
TZ applications can be secure-only

### LINKED ISSUE #203: trustzone applications can be secure-only
Hi

See https://github.com/Open-CMSIS-Pack/cmsis-toolbox/blob/main/docs/YML-Input-Format.md#processor

A secure application can be "secure" or "secure-only"

So at https://github.com/ARM-software/CMSIS_6/blob/main/ARM.CMSIS.pdsc#L624
Shouldn't we update:
```
<require Dsecure="Secure"/>
```
into?
```
  <accept Dsecure="Secure"/>
  <accept Dsecure="Secure-only"/>
```

### PR DESCRIPTION
**Pull request by jeromecoutant**
fix #203

### CODE REVIEW

File: ARM.CMSIS.pdsc [hunk 1]
```diff
@@ -621,7 +621,8 @@
     <condition id="TZ Secure">
       <description>TrustZone (Secure)</description>
       <require Dtz="TZ"/>
-      <require Dsecure="Secure"/>
+      <accept Dsecure="Secure"/>
+      <accept Dsecure="Secur-only"/>
```
Comment (jkrech) [inline ARM.CMSIS.pdsc:625]: spelling mistake.  It must read: "Secure-only"

**Review by jkrech — APPROVED** [review]
LGTM

File: ARM.CMSIS.pdsc
Comment (JonatanAntoni) [inline ARM.CMSIS.pdsc:1]: The scheme reference in line 3 needs update to version 1.7.40

Comment (jkrech) [reply ARM.CMSIS.pdsc:1]: Good catch.

**Review by jkrech — APPROVED** [review]
LGTM

**Pull request merged**